pkg/utils/resources.go

```diff
@@ -11,18 +11,29 @@ const (
 )
 
 func CalculateClusterResources(nodes *corev1.NodeList, pods *corev1.PodList) corev1.ResourceList {
-	base := GetNodesTotalResources(nodes)
-	reqs, _ := GetPodsTotalRequestsAndLimits(pods)
+	notNodes := GetNotReadyAndUnschedulableLeafNodes(nodes)
+	base := GetNodesTotalResources(nodes, notNodes)
+	reqs, _ := GetPodsTotalRequestsAndLimits(pods, notNodes)
 	podNums := GetUsedPodNums(pods, nodes)
 	SubResourceList(base, reqs)
 	SubResourceList(base, podNums)
 	return base
 }
 
-func GetNodesTotalResources(nodes *corev1.NodeList) (total corev1.ResourceList) {
+func GetNotReadyAndUnschedulableLeafNodes(leafNodes *corev1.NodeList) (notNodes map[string]corev1.Node) {
+	notNodes = make(map[string]corev1.Node)
+	for i, n := range leafNodes.Items {
+		if n.Spec.Unschedulable || !NodeReady(&leafNodes.Items[i]) {
+			notNodes[n.Name] = n
+		}
+	}
+	return notNodes
+}
+
+func GetNodesTotalResources(nodes *corev1.NodeList, notNodes map[string]corev1.Node) (total corev1.ResourceList) {
```

Comment:
```suggestion
func GetNodesTotalResources(nodes *corev1.NodeList, unavailableNodes map[string]corev1.Node) (total corev1.ResourceList) {
```
`notNodes` rename to `unavailableNodes` is better。

Reply:
I have fixed it